ci/prow/templates/testgrid_general_config.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright [[.Year]] The Knative Authors
```

Comment:
I think "testgrid_config_header.yaml" would be better.

Reply:
done